src/components/Kurlsh.js

```diff
@@ -240,6 +240,68 @@ class Kurlsh extends React.Component {
     };
   }
 
+  prepareVersions = versions => {
```

Comment:
possible to add tests? example here

https://github.com/replicatedhq/kurl.sh/blob/main/src/utils/kurl-yaml-test.js

Reply:
That took me a while to get it running. Gatsby seems to use `jest` for unit tests and I could not make the tests run with `mocha` as in the example you provided (my tests could not load the Kurl class file without exploding with an error saying that an "import" was not expected in a "non module", the error happened as soon as it attempted to load the Kurl class from the Kurlsh.js file). I looked in Gatsby documentation and added the necessary files to run the tests with jest instead. You can give it a try with `make test`, that will run both `mocha` and `jest` tests. 

Ideally we would not need to use two different tools but I would like to have someone with more experience than me to look into that (I have zero knowledge in any of the JS shenanigans) and I consider a miracle that I could get something working.

Reply:
Yes, the tests in `utils/kurl-yaml-test.js` were and are still are working just fine. I have tried to move the functions to another place, yes. The original error reported as soon as I tried to load the Kurl class from Kurlsh.js was:

```
SyntaxError: Cannot use import statement outside a module
```

And it was reporting that the error was on this [line](https://github.com/replicatedhq/kurl.sh/blob/main/src/components/Kurlsh.js#L1). When I thought about moving the code to a different place I realise that the error was going to be the same (we still need to import the `semver` package).

I am pretty sure this comes from my poor knowledge on the stack, but as gatsby [recommends jest](https://www.gatsbyjs.com/docs/how-to/testing/) anyways I decided to give it a try and the error vanished.

What do you reckon ?

Reply:
I'm fine with it. I don't know the stack either. Perhaps we can get a person who does to clean this all up in the future.